Dockerfile

```diff
@@ -1,18 +1,37 @@
-FROM quay.io/3scale/ruby:2.1
-MAINTAINER Oriol Martí <oriol@3scale.net> # 2015-06-26
+FROM centos:7 
+MAINTAINER Daniel Cesario <dcesario@redhat.com>
 
-RUN apt-install -y libssl-dev
+USER root
+
+RUN yum -y update \
+	&& yum install -y centos-release-scl \
+	&& yum install -y rh-ruby23 rh-ruby23-ruby-devel \
+	&& yum -y install make gcc \
+	&& yum -y install openssl-devel 
```

Comment:
Yes you are right, I was thinking on do it but I completely forgot to do so. My Bad I will update the PR.